src/test/isolation2/input/uao/vacuum_while_max_concurrency.source

```diff
@@ -0,0 +1,403 @@
+-- @Description Insert into a ao relation with 127 concurrent transactions
+-- 
+DROP TABLE IF EXISTS ao;
```

Comment:
The output of this query differs in the case of table exists andin the case of table doesn't exist. But it does not matter for the test. Let's ignore the query output.

Reply:
done